apps/election-manager/src/components/BallotCountsTable.tsx

```diff
@@ -272,6 +266,78 @@ const BallotCountsTable: React.FC<Props> = ({ breakdownCategory }) => {
         </Table>
       )
     }
+    case TallyCategory.VotingMethod: {
+      const resultsByVotingMethod =
+        fullElectionTally?.resultsByCategory.get(TallyCategory.VotingMethod) ||
+        {}
+      return (
+        <React.Fragment>
```

Comment:
Doesn't need to be wrapped in a `<React.Fragment>`, `<Table>` can be the top level tag here.